src/VisualStudio/VisualStudioInteractiveComponents/AssemblyRedirects.cs

```diff
@@ -5,17 +5,17 @@
 
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.Scripting.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.CSharp.Scripting.dll")]
-[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
+// [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
```

Comment:
What do you think is going to break here?  The VB scripting code was never loaded in this scenario.  Hence not registering it won't have an effect.